include/psp2kern/kernel/sysmem.h

```diff
@@ -203,6 +203,10 @@ int ksceKernelGetPaddr(void *addr, uintptr_t *paddr);
 
 int ksceSysrootIsManufacturingMode(void);
 
+int ksceDebugPrintf(const char *fmt, ...);
+
+int ksceDebugPrintf2(int unk0, int unk1, const char *fmt, ...);
```

Comment:
I think she just got a NID info & reverse info, and only use dumping log from kernel
https://github.com/Cpasjuste/psp2shell/blob/master/psp2shell_k/main.c#L82-L86
so I though, she can't know mean of args.